openhands/runtime/impl/action_execution/action_execution_client.py

```diff
@@ -264,6 +265,12 @@ def send_action_for_execution(self, action: Action) -> Observation:
                 raise AgentRuntimeTimeoutError(
                     f'Runtime failed to return execute_action before the requested timeout of {action.timeout}s'
                 )
+            except requests.HTTPError as e:
```

Comment:
I think we decided to move this to the remote runtime?

Reply:
oh yes! removed -- forget to clean this up. good catch!